<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmMain</class>
 <widget class="QMainWindow" name="frmMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>340</width>
    <height>200</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>QNapi - program do pobierania napisów do filmów</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res/resources.qrc">
    <normaloff>:/icon/qnapi.png</normaloff>:/icon/qnapi.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 4px dashed gray;
margin: 15px;</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="acceptDrops">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">border:0; color: gray; margin:0;</string>
         </property>
         <property name="text">
          <string>Upuść pliki/katalogi tutaj</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuNapisy">
    <property name="title">
     <string>Napisy</string>
    </property>
    <widget class="QMenu" name="menuNapiProjekt">
     <property name="title">
      <string>NapiProjekt</string>
     </property>
     <addaction name="actionNPDownload"/>
     <addaction name="actionNPConfigure"/>
     <addaction name="actionNPRegister"/>
    </widget>
    <widget class="QMenu" name="menuOpenSubtitles">
     <property name="title">
      <string>OpenSubtitles</string>
     </property>
     <addaction name="actionOSDownload"/>
     <addaction name="actionOSConfigure"/>
     <addaction name="actionOSRegister"/>
    </widget>
    <widget class="QMenu" name="menuNapisy24">
     <property name="title">
      <string>Napisy24</string>
     </property>
     <addaction name="actionN24Download"/>
     <addaction name="actionN24Configure"/>
     <addaction name="actionN24Register"/>
    </widget>
    <addaction name="actionSubtitleDownload"/>
    <addaction name="actionDirectoryScan"/>
    <addaction name="actionConvert"/>
    <addaction name="separator"/>
    <addaction name="menuNapiProjekt"/>
    <addaction name="menuOpenSubtitles"/>
    <addaction name="menuNapisy24"/>
   </widget>
   <widget class="QMenu" name="menuProgram">
    <property name="title">
     <string>Program</string>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="actionAbout"/>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuProgram"/>
   <addaction name="menuNapisy"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSubtitleDownload">
   <property name="text">
    <string>Pobierz</string>
   </property>
  </action>
  <action name="actionDirectoryScan">
   <property name="text">
    <string>Skanuj katalogi</string>
   </property>
  </action>
  <action name="actionConvert">
   <property name="text">
    <string>Konwertuj</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Ustawienia</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>O programie</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Zakończ</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Pomoc</string>
   </property>
  </action>
  <action name="actionNPDownload">
   <property name="text">
    <string>Pobierz</string>
   </property>
  </action>
  <action name="actionNPConfigure">
   <property name="text">
    <string>Skonfiguruj</string>
   </property>
  </action>
  <action name="actionNPRegister">
   <property name="text">
    <string>Załóż konto</string>
   </property>
  </action>
  <action name="actionOSDownload">
   <property name="text">
    <string>Pobierz</string>
   </property>
  </action>
  <action name="actionOSConfigure">
   <property name="text">
    <string>Skonfiguruj</string>
   </property>
  </action>
  <action name="actionOSRegister">
   <property name="text">
    <string>Załóż konto</string>
   </property>
  </action>
  <action name="actionN24Download">
   <property name="text">
    <string>Pobierz</string>
   </property>
  </action>
  <action name="actionN24Configure">
   <property name="text">
    <string>Skonfiguruj</string>
   </property>
  </action>
  <action name="actionN24Register">
   <property name="text">
    <string>Załóż konto</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../res/resources.qrc"/>
 </resources>
 <connections/>
</ui>
